docs/Validation-and-Serialization.md

```diff
@@ -498,3 +500,6 @@ const refToSharedSchemaDefinitions = {
 - [Understanding JSON schema](https://spacetelescope.github.io/understanding-json-schema/)
 - [fast-json-stringify documentation](https://github.com/fastify/fast-json-stringify)
 - [Ajv documentation](https://github.com/epoberezkin/ajv/blob/master/README.md)
+- [Ajv i18n](https://github.com/epoberezkin/ajv-i18n)
+- [Ajv custom errors](https://github.com/epoberezkin/ajv-errors)
+- Custom error handling with core methods with error file dumping [example](https://gist.github.com/dalisoft/a6a088860e724996d0baecae59ace329)
```

Comment:
Sorry, but I don't think it is a nice idea to link an external gist in docs.

I'll try to push the example in the [example repo](https://github.com/fastify/example/)

Reply:
Ok, thanks :)